.travis.yml

```diff
@@ -0,0 +1,35 @@
+language: java
+jdk:
+- openjdk11
+branches:
+  only:
+  # Since we're using conventional commit paradigm, we only want a master merge commit to trigger a build: https://www.conventionalcommits.org/en/v1.0.0-beta.4/
+  - master
+env:
+  global:
+  # This environment variable is to support the rollover from the initial beta artifact release
+  - BETA_TAG="-beta"
+before_script:
+  # Before updating versioning and pushing artifacts, copy over the maven settings file with ossrh and gpg credentials
+  - cp .travis.settings.xml $HOME/.m2/settings.xml
+script:
+  # Find the most recent version via git tags
+  - OLD_VERSION=$(npx git-semver-tags | head -n 1)
+  # Using conventional commits paradigm, check the merge commit message on master to set up a new version deployment
+  - if [ -n "$OLD_VERSION" ]; then
```

Comment:
Another thing to consider is to break this job into two jobs/stages and pass the version as an artifact between teh two.

Reply:
I like the idea of having everything in one pipeline. It makes it a little easier to manage imo.

Reply:
I see what you mean regarding the jobs for the pipeline. I'm experimenting with that now